src/Tribe/Editor/Meta.php

```diff
@@ -194,7 +194,19 @@ public function meta_backwards_compatibility( $data, $post, $request ) {
 	public function update_cost( $post_data, $request ) {
 		$post_id = $request->get_param( 'id' );
 
-		\Tribe__Events__API::update_event_cost( $post_id );
+		// Fetch cost data from the request (if set).
+		$json = $request->get_json_params();
+
+		// Set an appropriate default for the cost.
+		$default = (array) tribe_get_cost( $post_id );
```

Comment:
Suggestion: I'd use `$cost` or something similar, rather than `$default`, since we may change it (and then it's no longer the default?)

Probably above as well 🤔

Reply:
@Camwyn I've updated this to $cost